waiter/config-full.edn

```diff
@@ -475,4 +475,10 @@
 
  ;; Waiter launches a loop that maintains the stacktraces for individual threads;
  ;; this value dictates how often (milliseconds) the stacktraces are refreshed:
- :thread-stack-state-refresh-interval-ms 600000}
+ :thread-stack-state-refresh-interval-ms 600000
+
+ ;; Waiter supports making websocket connections to backends.
+ ;; The maximum binary and text message sizes in the websocket request or response can be configured
+ ;; as follows, the values are in bytes:
+ :websocket-config {:ws-max-binary-message-size  137438953472
```

Comment:
These are really big limits for a default config.  Should the default config be more reasonable?  Is there a reason for the numbers you chose that you can comment on?